src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkContentTests/MsftToSbSdkFiles.diff

```diff
@@ -44,11 +48,554 @@ index ------------
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.CoreLib.xml
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.DisabledReflection.dll
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.DisabledReflection.pdb
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.DisabledReflection.xml
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.Reflection.Execution.dll
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.Reflection.Execution.pdb
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.Reflection.Execution.xml
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.StackTraceMetadata.dll
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.StackTraceMetadata.pdb
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.StackTraceMetadata.xml
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.TypeLoader.dll
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.TypeLoader.pdb
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/sdk/System.Private.TypeLoader.xml
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/ilc
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/libclrjit_universal_arm_x64.so
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/libclrjit_universal_arm64_x64.so
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/libclrjit_unix_x64_x64.so
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/libclrjit_win_x64_x64.so
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/libclrjit_win_x86_x64.so
++./packs/runtime.banana-rid.Microsoft.DotNet.ILCompiler/x.y.z/tools/libjitinterface_x64.so
+ ./sdk-manifests/
+ ./sdk-manifests/x.y.z/
+ ./sdk-manifests/x.y.z/
 @@ ------------ @@
  ./sdk/x.y.z/Microsoft.Build.NuGetSdkResolver.dll
  ./sdk/x.y.z/Microsoft.Build.Tasks.Core.dll
  ./sdk/x.y.z/Microsoft.Build.Utilities.Core.dll
 -./sdk/x.y.z/Microsoft.CodeCoverage.IO.dll
  ./sdk/x.y.z/Microsoft.Common.CrossTargeting.targets
  ./sdk/x.y.z/Microsoft.Common.CurrentVersion.targets
- ./sdk/x.y.z/Microsoft.Common.overridetasks
\ No newline at end of file
+ ./sdk/x.y.z/Microsoft.Common.overridetasks
+@@ ------------ @@
+ ./sdk/x.y.z/Roslyn/bincore/ru/Microsoft.CodeAnalysis.CSharp.resources.dll
+ ./sdk/x.y.z/Roslyn/bincore/ru/Microsoft.CodeAnalysis.resources.dll
+ ./sdk/x.y.z/Roslyn/bincore/ru/Microsoft.CodeAnalysis.VisualBasic.resources.dll
++./sdk/x.y.z/Roslyn/bincore/System.Collections.Immutable.dll
++./sdk/x.y.z/Roslyn/bincore/System.Reflection.Metadata.dll
```

Comment:
Reopened https://github.com/dotnet/source-build/issues/4016

Reply:
Will readd exclusion and resolve with https://github.com/dotnet/source-build/issues/4016